lib/locale/pl_PL/strings.php

```diff
@@ -24,6 +24,7 @@
  *  $Id$
  */
 
+$_LANG['bank loan'] = 'pożyczka bankowa';
 $_LANG['starting with'] = 'rozpoczynające się od';
```

Comment:
Na początku umieszczamy tłumaczenia, które są kandydatami do przeniesienia do stable-ów - dla uproszczenia.

Reply:
O, nie wiedziałem tego. 
Wiesz co? W sumie tą zmianę można by wrzucić do stable.

Reply:
Tak to chyba dobry pomysł.